proposals/incubation/istio.md

```diff
@@ -0,0 +1,375 @@
+### Name of the project
+
+Istio
+
+### Description
+
+Istio is an open source service mesh that transparently provides a uniform and efficient way to secure, connect, and monitor services in cloud native applications. It provides zero-trust networking, policy enforcement, traffic management, load balancing, and monitoring, without requiring applications to be rewritten.
+
+Features include:
+
+* Secure service-to-service communication based on zero-trust principles, using TLS encryption and strong identity-based authentication and authorization
+* Automatic load balancing for HTTP, gRPC, WebSocket, and TCP traffic
+* Fine-grained control of traffic behavior with rich routing rules, retries, failovers, and fault injection
+* A policy API supporting access controls, rate limits and quotas, that can be connected to external authorization systems
+* Automatic generation of metrics, logs, and traces for all traffic within a cluster, including cluster ingress and egress
+* A data plane platform allowing for extension and customization of proxy behavior
+
+Istio is designed for extensibility, and can handle a diverse range of deployment needs. Istio’s control plane runs on Kubernetes, and is capable of managing workloads running in the same Kubernetes cluster, other Kubernetes clusters, and even workloads running outside Kubernetes on VMs.
+
+### Sponsor / Advisor from TOC
+
+To be appointed.
```

Comment:
Thanks for your application and welcome to CNCF.

the TOC team will figure out who can help with the incubation process and let you all know

Reply:
@craigbox I'll be your sponsor for this process.
Let's connect next week to figure out next steps and get this moving.

Reply:
Thank you Dave for the sponsorship! Steering is looking forward to working with you on this. Please use istio-steering@googlegroups.com  for comms so we're able to track all the moving parts!

Reply:
I can also co-sponsor if you need help @dzolotusky :)